.circleci/config.yml

```diff
@@ -1,63 +1,91 @@
-version: 2
+version: 2.1
+
+orbs:
+  python: circleci/python@1.2.0
 
-workflows:
-  version: 2
-  test:
-    jobs:
-      - build-python-3.6
-      - build-python-3.7
-      - build-python-3.8
-      - build-python-3.9
 
 jobs:
-  build-python-3.6: &template
+  pre-build:
+    description: Install and run a Python standalone package
+    parameters:
+      command-name:
+        type: string
+      command-run:
+        type: string
     docker:
-      - image: python:3.6
+      - image: cimg/python:3.7
+        #auth:
+        #  username: $DOCKERHUB_USERNAME
+        #  password: $DOCKERHUB_PASSWORD
```

Comment:
I think we can safely remove these lines. I don't envision the need for passing in Docker Hub credentials for this project.

Reply:
Starting from 1st November, CircleCI will limit the unauthenticated Docker pulls, you can read more [here](https://circleci.com/docs/2.0/private-images/).

Reply:
I see -- thanks for flagging this! In that case, I agree it's good to leave these commented-out lines here as a reminder to ourselves.